src/gui/ar/groundPlaneAnchors.js

```diff
@@ -177,26 +136,28 @@ createNameSpace("realityEditor.gui.ar.groundPlaneAnchors");
     // helper function to create the geometry for a surface anchor, including its X-Z axis handles
     function createAnchorGroup(frameKey) {
         const THREE = realityEditor.gui.threejsScene.THREE;
-        const group = new THREE.Group();
+
+        let originSize = 100;
```

Comment:
It would be nice to have this origin box scale the same way as the transform controls (it looks like they're a constant size regardless of distance). Alternatively vice-versa and have the transform controls a fixed size. Totally just an optional polish thing that isn't necessary if it'll be a pain to do

Reply:
I gave it a shot and it worked on remote operator but for some reason it works differently in the AR devices so I'm going to leave it out for now.